backend/helper_zero/serializers.py

```diff
@@ -1,18 +1,19 @@
 from rest_framework import serializers
-from .models import Organization
-from .models import DonationRequest
+from .models import User, Organization, DonationRequest
+
+class UserSerializer(serializers.ModelSerializer):
```

Comment:
oh also you'll need to specify that zipcode, lat, and lon could be blank. Looks like this:
```
zipcode = serializers.CharField(allow_null=True)
```

Reply:
Can you test creating a user locally with only one of them set?

Reply:
Resolved in person